src/movies/Movies.js

```diff
@@ -1,39 +1,37 @@
 import React, { Component } from 'react';
+import { getAllMovies } from '../apiCalls'
 import "./Movies.css"
 
-class Movies extends Component {
+export default class Movies extends Component {
   constructor() {
     super();
     this.state = {
-      movies:[],
-    }
+      movies: [],
+    };
   }
 
-  getMovies = () => {
-    fetch("https://rancid-tomatillos.herokuapp.com/api/v2/movies")
-    .then(response => response.json())
+  componentDidMount () {
+    getAllMovies()
     .then(data => this.setState({ movies: data.movies }))
     .catch(error => this.setState({ error, isLoading: false}));
   }
-  moviesToDisplay = this.state.movies.map((movie, index)=> {
-    return (
-      <article key={index} className='poster-card' id={movie.id}>
-        <img className='image-poster' src={movie.poster_path}></img>
-        <section className='poster-card-text'>
-          <h2 className='title-poster'>{movie.title}</h2>
-          <h3 className='rating-poster'>Average Rating: {movie.average_rating}</h3>
-          <h3 className='release-date-poster'>Release Date: {movie.release_date}</h3>
-        </section>
-      </article>
-    )
-  })
+
   render () {
     return (
-      <section className="movies-list">
-        {this.moviesToDisplay}
-      </section>
+        <section className="movies-list" title="movies-list">
+          { this.state.movies.map((movie, index) => {
+            return (
+              <article key={index} className='poster-card' title='poster-card' id={movie.id}>
+                <img alt='image-poster' src={movie.poster_path}></img>
+                <section className='poster-card-text'>
+                  <h2 className='title-poster'>{movie.title}</h2>
+                  <h3 className='rating-poster'>Average Rating: {movie.average_rating}</h3>
+                  <h3 className='release-date-poster'>Release Date: {movie.release_date}</h3>
+                </section>
+              </article>)
+            })
+          }
```

Comment:
I wouild recommend taking this back out of the render statement and making it a variable or property that can be referred to, like it was before. We want the `render` method to be singularly responsible for returning UI, not iterating over data.